docs/tutorials/demo.ipynb

```diff
@@ -43,35 +43,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "id": "10f6c9ed-fe58-4e03-b29b-c6c447061f84",
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "'0.3.0'"
-      ]
-     },
-     "execution_count": 1,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "import earthaccess\n",
     "earthaccess.__version__"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "id": "2acb1926-3442-446a-8a35-fcbba8868d4d",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from earthaccess import Auth, Store, DataCollections, DataGranules\n",
-    "auth = Auth()"
+    "auth = earthaccess.login(strategy=\"environment\")"
```

Comment:
Could we move this below the next markdown block? That way we can give the full explanation of all of the different auth strategies and then actually execute them (otherwise it's a bit out of order to start with the environment strategy, then explanation, then netrc.